zerver/actions/user_groups.py

```diff
@@ -67,18 +67,6 @@ def create_user_group_in_database(
             event_time=creation_time,
             modified_user_group=user_group,
         ),
-        RealmAuditLog(
-            realm=realm,
-            acting_user=acting_user,
-            event_type=RealmAuditLog.USER_GROUP_GROUP_BASED_SETTING_CHANGED,
-            event_time=creation_time,
-            modified_user_group=user_group,
-            extra_data={
-                RealmAuditLog.OLD_VALUE: None,
-                RealmAuditLog.NEW_VALUE: user_group.can_mention_group.id,
-                "property": "can_mention_group",
-            },
-        ),
```

Comment:
This was added in 3349ac9f86e7d5b40a26dd2f6b93b165b24511e3; @PIG208 FYI in case you remember why we wanted these initial values to be present in `RealmAuditLog`. I agree with the PR author that we usually do not create audit log entries for initial values of fields.

Reply:
We discussed this [here](https://chat.zulip.org/#narrow/stream/101-design/topic/New.20permissions.20model/near/1633816).

Reply:
The motivation was [explained here](https://chat.zulip.org/#narrow/stream/101-design/topic/New.20permissions.20model/near/1613344). It is not strictly necessary because if `can_mention_group` has never been changed, we can just get its "NEW_VALUE" directly.

> Zixuan James Li: It exists there so that the current value of can_mention_group always has an audit log with a matching NEW_VALUE.

> Zixuan James Li: This way, before can_mention_group is ever changed, we still have an audit log of how the initial value came about.

> Zixuan James Li: Alternatively, we can track that setting in the creation event and populate extra data without reusing the USER_GROUP_GROUP_BASED_SETTING_CHANGED event. I guess this might make more sense to avoid violating the null constraint?

Reply:
I think because we log the OLD_VALUE and NEW_VALUE, the audit logs would already have all the historical values of the setting, so I think we're good to proceed with these changes as planned.